nginx_stage/etc/profile

```diff
@@ -8,4 +8,9 @@
 #
 SCL_PKGS=${SCL_PKGS:-"ondemand"}
 SCL_SOURCE="$(command -v scl_source)"
-[[ "${SCL_SOURCE}" ]] && source "${SCL_SOURCE}" enable ${SCL_PKGS} || export GEM_PATH="/opt/ood/gems:$GEM_PATH"
+if [[ "${SCL_SOURCE}" ]]; then
+  source "${SCL_SOURCE}" enable ${SCL_PKGS}
+else
+  export GEM_PATH="/opt/ood/gems:$GEM_PATH"
+  export RUBYLIB="/opt/ood/ondemand/root/usr/share/ruby/vendor_ruby:${RUBYLIB:+:${RUBYLIB}}"
```

Comment:
This is normally set by the SCL enable code so this is to ensure proper paths for Ubuntu.

I wonder though if should instead create like `/opt/ood/profile` with these items in it for just Ubuntu and source that file.  Could be sourced by other profiles if ever became needed.  Could go so far as to mimic SCL and create `opt/ood/ondemand/enable` which is what SCL uses to setup the environment.

Reply:
> I wonder though if should instead create like /opt/ood/profile

Likely. ood-portal-generator has similar logic. I'd be up for one, doesn't have to be this PR.

https://github.com/OSC/ondemand/blob/master/ood-portal-generator/etc/profile

I think the documentation would read the same, just a different source file.

https://osc.github.io/ood-documentation/latest/customization_overview.html?highlight=etc%20ood%20profile

Reply:
The `/etc/ood/profile` is for admin defined values.  I'll go ahead and make the necessary adjustment in this PR, as I had the changes staged just didn't commit yet.